core/experiments/assets/js/admin/behaviors/experiments-messages.js

```diff
@@ -109,44 +118,35 @@ export default class ExperimentsDependency {
 		return ! this.areAllDependenciesActive( dependencies );
 	}
 
-	showDependenciesDialog( experimentId ) {
-		const experiment = this.getExperimentData( experimentId ),
-			dependencies = this.getExperimentDependencies( experimentId );
+	shouldShowDeactivationDialog( experimentId ) {
+		const initialState = this.getExperimentData( experimentId ).state,
```

Comment:
maybe use experimentData= this.getExperimentData( experimentId );
and then access the state and default attributes instead of calling the function twice?

Reply:
Done :)